magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -409,28 +351,29 @@ def mc_dl0_to_dl1(input_file, output_dir, config):
         logger.info(f"\nIn total {n_events_processed} events are processed.")
 
     # Convert the telescope coordinate to the one relative to the center
-    # of the LST-1 and MAGIC positions, and reset the telescope IDs
+    # of the LST and MAGIC positions, and reset the telescope IDs
     position_mean = u.Quantity(list(tel_positions.values())).mean(axis=0)
```

Comment:
can't you use telescope_positions function from the ...coincidence macro?

Reply:
Do you mean this?
`RELATIVE_POSITIONS = {
    "LST-1" : [-70.930, -52.070, 43.00],
    "LST-2" : [-35.270,  66.140, 32.00],
    "LST-3" : [75.280 ,  50.490, 28.70],
    "LST-4" : [30.910 , -64.540, 32.00],
    "MAGIC-I" : [-23.540, -191.750, 41.25],
    "MAGIC-II" : [-94.05, -143.770, 42.42]
    }`
These are the telescope positions (for the 6 telescopes) in a custom reference system. Both in MC DL0 to DL1and coincidence macro we 're-compute' these positions for the used telescopes only (e.g., LST1 + MAGICs) in a reference system centered in the center of the system of the used telescopes. In MCs the input to recompute them are the informations saved in MC files, while in coincident events we use the 'hard-coded' dictionary (but the telescope relative distances are the same in MCs and real data, see image for an LSTProd2 file telescope coordinates). So we cannot import the dictionary from the coincidence script as it is, but any suggestion on how to optimize this is welcome!

![image](https://github.com/cta-observatory/magic-cta-pipe/assets/121040436/ccab15bd-0847-4b76-922f-4876dfeb9d48)

Reply:
I've meant the recomputing part, because it is basically the same, but just with different inputs, but ok, we can keep it as it is now and might improve it in the future